### ISSUE TITLE
[desktop_window_webview] When closing webview window, it also closes main window on Linux

### ISSUE THREAD

**Issue by MuhammedKpln** [MixinNetwork/flutter-plugins#251]
When closing the webview window created by desktop_window_webview, it also closes the main window.  Which it should'nt do it.

## Expected behavior

When closing webview window, it should not interrupt the main window.

```
Flutter 3.10.1 • channel stable • https://github.com/flutter/flutter.git
Framework • revision d3d8effc68 (6 days ago) • 2023-05-16 17:59:05 -0700
Engine • revision b4fb11214d
Tools • Dart 3.0.1 • DevTools 2.23.1

```

**Comment by ZG-123** [comment]
Hi, I also met the same problem, do not know how to avoid

**Comment by wanghongenpin** [comment]
+1

**Comment by boyan01** [comment]
this should be fixed in version 0.2.1. closing 

https://pub.dev/packages/desktop_webview_window/changelog


feel free to reopen if not fixed.

**Comment by wanghongenpin** [comment]
desktop_multi_window  啥时候发布呢

> this should be fixed in version 0.2.1. closing
> 
> https://pub.dev/packages/desktop_webview_window/changelog
> 
> feel free to reopen if not fixed.
